### PR TITLE
[CTCP-5552] added incident details page

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

**Comment by prashanthmamidi** [comment]
image

File: conf/application.conf [hunk 1]
```diff
@@ -112,7 +112,7 @@ microservice {
     features {
       isGuaranteeBalanceEnabled = true
       isPhase4Enabled = true
-      isPhase5Enabled = false
+      isPhase5Enabled = true
```
Comment (kevincoatsworth) [inline conf/application.conf:115]: Should this be reverted to false?

File: test/base/SpecBase.scala [hunk 2]
```diff
@@ -44,7 +44,8 @@ trait SpecBase extends AnyFreeSpec with Matchers with OptionValues with TryValue
   val departureIdP5: String    = "643cffea2dca70b2"
   val arrivalIdP5: String      = "62f4ebbb765ba8c2"
 
-  val incidentIndex: Int = 0
+//  val incidentIndex: Int = 0  // TODO
```
Comment (kevincoatsworth) [inline test/base/SpecBase.scala:47]: Guessing this can be removed

File: conf/application.conf (hunk 1 above)
Comment (prashanthmamidi) [reply conf/application.conf:115]: yes

**Review by kevincoatsworth — APPROVED** [review]

**Pull request merged**